### ISSUE TITLE
registerMenuCommand text label does not appear in the Toolbar Button menu

### ISSUE THREAD

**Issue by darkconsole** [Tampermonkey/tampermonkey#1506]
the text from GM_registerMenuCommand is not currently appearing in the toolbar menu button. works fine if you click it. just missing the text.

![image](https://user-images.githubusercontent.com/9095897/167520271-23d018b7-2824-4125-924b-e14565f940f9.png)

but it does appear in the right click context menu.

![image](https://user-images.githubusercontent.com/9095897/167520323-0b13c7c9-5f89-48b3-9d08-be7b79bad218.png)

## Specifications

- Firefox: Mozilla/5.0 (Windows NT 10.0; Win64; x64; rv:100.0) Gecko/20100101 Firefox/100.0
- TM: 4.16.6160
- OS: Windows 10 Pro 21H2

## Script

```js
// ==UserScript==
// @name         Fuck Your WebP
// @version      1
// @namespace    darkcee.net
// @description  try to break CDNs that are sending you WebP when the URL claims it should not be one.
// @updateURL    https://darkcee.net/code/tampermonkey/FuckYourWebP.js
// @downloadURL  https://darkcee.net/code/tampermonkey/FuckYourWebP.js
// @homepage     https://darkcee.net/code/browser-userscripts
// @icon         https://darkcee.net/favicon.ico
// @author       darkconsole

// @match        http://*/*
// @match        https://*/*

// @noframes
// @grant        GM_registerMenuCommand
// ==/UserScript==

(function() {
	'use strict';

	class FuckYourWebP {

			constructor() {
				this.queryVariable = 'webp';
				this.queryValue = 'fuckingstupid';
				return;
			};

			yeahFuckIt() {

				if(location.href.match(/\?/))
				location.href = `${location.href}&${this.queryVariable}=${this.queryValue}`;

				else
				location.href = `${location.href}?${this.queryVariable}=${this.queryValue}`;

				return;
			};

	};

	////////////////////////////////////////////////////////////////
	////////////////////////////////////////////////////////////////

	let webp = new FuckYourWebP;

	GM_registerMenuCommand(
		'Fuck Your WebP',
		webp.yeahFuckIt.bind(webp)
	);

	return;
})();
```

**Comment by derjanb** [comment]
Should be fixed at 4.18.6163 (in review|[crx](https://tampermonkey.net/crx/tampermonkey_beta_current.crx)|[xpi](https://firefox.tampermonkey.net/firefox-current-beta.xpi))

Chrome/Edge users, please download the crx file linked above and drag and drop it to the extensions page `chrome://extensions`.
Firefox users please install the [BETA version](https://www.tampermonkey.net/index.php?browser=firefox) or check for BETA version updates at `about:addons` 

For a quick fix please export your settings and scripts as zip or (JSON) file at the "Utilities" tab and import it back at the fixed BETA version.

**Comment by darkconsole** [comment]
looks good on this end, cheers m8.
